src/main/java/io/syndesis/qe/marketplace/manifests/Opm.java

```diff
@@ -0,0 +1,81 @@
+package io.syndesis.qe.marketplace.manifests;
+
+import io.syndesis.qe.marketplace.quay.QuayUser;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+@Slf4j
+public class Opm {
+
+    private File binary;
+    private static final String OPM_IMAGE = "registry.redhat.io/openshift4/ose-operator-registry:";
+    private static final String version = "v4.5";
+
+    public Opm() {
+        try {
+            Process which = new ProcessBuilder("which", "opm").start();
+            int result = which.waitFor();
+            String path = IOUtils.toString(which.getInputStream(), Charset.defaultCharset()).trim();
+            if (result != 0) {
+                fetchOpm(version);
+            } else {
+                binary = new File(path);
+            }
+        } catch (InterruptedException | IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private Opm(String version) {
+        fetchOpm(version);
+    }
+
+    private void fetchOpm(String version) {
+        try {
+            binary = File.createTempFile("opm", version);
+            Process p = new ProcessBuilder("docker", "run", "--rm", "--entrypoint=/usr/bin/cat", OPM_IMAGE + version, "/usr/bin/opm")
+                    .redirectOutput(binary)
+                    .start();
+            p.waitFor();
+            if (p.exitValue() != 0) {
+                log.error(IOUtils.toString(p.getErrorStream(), Charset.defaultCharset()));
+            }
+            binary.setExecutable(true);
+            //binary.deleteOnExit();
+        } catch (IOException | InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @SneakyThrows
+    int runCmd(String... args) {
```

Comment:
don't you want to use one method for all commands invocations? It seems to me that this code is doing the same as `Index#cmdWrapper` and it could be also used in `Bundle#metadata`